### PR TITLE
SDK-1005: yoti hub rename

### PR DESCRIPTION
**Pull request by emmas-yoti**
### Changed
 - README.md updated with new Yoti Hub branding, URI references updated to point to new hub.yoti.com site

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ timestamp = source_anchor.signed_timestamp
 origin_server_certs = source_anchor.origin_server_certs
 ```
 
-If you have chosen `Verify Condition` on the [Yoti Dashboard](https://www.yoti.com/dashboard/applications) with the age condition of "Over 18", you can retrieve the user information as follows:
+If you have chosen `Verify Condition` on the [Yoti Hub](https://hub.yoti.com/applications) with the age condition of "Over 18", you can retrieve the user information as follows:
```
Comment (davidgrayston) [inline README.md]: I think `/applications` behaves differently for organisations - I changed this to just `https://hub.yoti.com` in other SDKs - thoughts @echarrod ?

Comment (echarrod) [reply README.md]: Yep, `/applications` is personal applications, let's drop it

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Please feel free to reach out
 ## An Architectural View
 
 Before you start your integration, here is a bit of background on how the integration works. To integrate your application with Yoti, your back-end must expose a GET endpoint that Yoti will use to forward tokens.
-The endpoint can be configured in the Yoti Dashboard when you create/update your application. For more information on how to create an application please check our [developer page](https://www.yoti.com/developers/documentation/#login-button-setup).
+The endpoint can be configured in the Yoti Hub when you create/update your application. For more information on how to create an application please check our [developer page](https://www.yoti.com/developers/documentation/#login-button-setup).
```
Comment (echarrod) [inline README.md]: This is slightly different now that we've moved the documentation, I'll make a ticket to look through the SDKs to check these links
```suggestion
The endpoint can be configured in the Yoti Hub when you create/update your application. For more information on how to create an application please check our [developer page](https://developers.yoti.com/yoti-app-integration/yoti-app-integration#step-3-front-end-integration).
```

File: README.md [hunk 3]
```diff
@@ -173,11 +173,11 @@ Yoti will provide a boolean result on the following checks:
 * Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
 * Watch list - Verify against watch lists from the Office of Foreign Assets Control
 
-To use this functionality you must ensure your application is assigned to your Organisation in the Yoti Dashboard - please see [here](https://www.yoti.com/developers/documentation/#1-creating-an-organisation) for further information.
+To use this functionality you must ensure your application is assigned to your Organisation in the Yoti Hub - please see [here](https://www.yoti.com/developers/documentation/#1-creating-an-organisation) for further information.
```
Comment (echarrod) [inline README.md]:
```suggestion
To use this functionality you must ensure your application is part of an Organisation in the Yoti Hub - please see [here](https://developers.yoti.com/yoti-app-integration/yoti-app-integration#step-1-creating-an-organisation) for further information.
```

File: README.md [hunk 4]
```diff
@@ -222,7 +222,7 @@ Additionally an [example AML application](/examples/aml/app.py) is provided in t
 
 ## Running the Examples
 
-From the [Yoti Dashboard](https://www.yoti.com/dashboard/applications):
+From the [Yoti Hub](https://hub.yoti.com/applications):
```
Comment (echarrod) [inline README.md]:
```suggestion
From the [Yoti Hub](https://hub.yoti.com):
```

File: README.md (hunk 1 above)
Comment (emmas-yoti) [inline README.md]:
```suggestion
If you have chosen `Verify Condition` on the [Yoti Hub](https://hub.yoti.com) with the age condition of "Over 18", you can retrieve the user information as follows:
```

**Pull request merged**